api/src/i18n/services/translation.service.spec.ts

```diff
@@ -58,12 +67,16 @@ describe('TranslationService', () => {
         {
           provide: SettingService,
           useValue: {
-            getSettings: jest.fn().mockResolvedValue({
-              chatbot_settings: {
-                global_fallback: true,
+            getSettings: jest
+              .fn()
+              .mockResolvedValue(['Global fallback message']),
+            find: jest.fn().mockResolvedValue([
+              {
+                translatable: false,
+                global_fallback: 'global_fallback',
```

Comment:
```suggestion
                label: 'global_fallback',
```
Focus is the bridge between intention and accomplishment 🙏

Reply:
Done